### PR TITLE
Song: Add support for Yarock and Qmmp

### PR DESCRIPTION
**Pull request by mstraube**
- Adds support for Yarock and Qmmp.
- Generally don't print empty song info `" - "`.
This way we don't have to care about that for every single player.

### CODE REVIEW

File: neofetch [hunk 1]
```diff
@@ -1266,6 +1268,8 @@ get_song() {
         ;;
     esac
 
+    [[ "$song" =~ ^[\ -]+$ ]] && song=""
```
Comment (dylanaraps) [inline neofetch]: I don't think we need regex matching here.

Does this work?:

```sh
[[ "$song" == " - " ]] && unset -v song

# Safer alternative
[[ "$(trim "$song")" == " - " ]] && unset -v song
```

**Comment by dylanaraps** [comment]
Other than my comment, LGTM! :+!:

File: neofetch (hunk 1 above)
Comment (mstraube) [inline neofetch]: I'm not sure, I'll do some test...

**Comment by mstraube** [comment]
It's `[[ "$(trim "$song")" == "-" ]] && unset -v song`

I also changed the query command for deadbeef. `--nowplaying` is deprecated and sets song to `"nothing"` if playback is stopped.

All good now.

**Comment by dylanaraps** [comment]
:+1:

**Pull request merged**